### DISCUSSION TITLE
Getting 404 when trying to access traefik login on kubernetes.

### DISCUSSION THREAD

**Discussion by alteredtech** [timothystewart6/launchpad#66] [Questions] [answered]
tl;dr: try chrome.

Not so much a question but an answer to a question I was looking for.
When following the traefik and cert-manager tutorial and get to the part to try going to the traefik dashboard with the traefik cert. You will be prompted to agree to the cert and then met with a 404 page. There is probably nothing wrong. 

I was going over all my files with a fine tooth pick to see if I accidentally messed something up. None. Just a browser issue
After getting 404 on firefox and safari on my laptop. I decided to try firefox (aka safari with a skin) on my iphone. And sure enough, the login popup showed up. I then went back to my computer and disabled the popup warnings and autodisable popups, still nothing on safari and firefox. Also tried private browser with no luck.

Then I remembered that Tim was using chrome. Sure enough, login popup showed right up. Something funky with safari and firefox on mac/windows.  

If anyone knows of a setting or something to allow safari and firefox to show the login popup, I would love to know. Until then going to stick with my phone and a chrome wishing it was the default browser.

**Comment by alteredtech** [comment, marked as answer]
tl;dr: try chrome.
